### PR TITLE
updated to work with docker compose v2

### PR DESCRIPTION
**Pull request by siddhanthrathod**
# Description
  Used the try/catch to work with docker compose v2.

### CODE REVIEW

File: start.py [hunk 1]
```diff
@@ -222,17 +222,30 @@ def start():
...
+                        env["BUILDKIT_PROGRESS"] = "plain"
+                subprocess.run(command, env=env, check=True)
+        #for docker compose v2
+        except OSError :
+                command = ["docker","compose"] + base_command[1:] + [args.docker_command] + unknown
+                env = os.environ.copy()
+                env["DOCKER_BUILDKIT"] = "1"
+                if args.debug_build:
+                        env["BUILDKIT_PROGRESS"] = "plain"
+                subprocess.run(command, env=env, check=True)
```
Comment (mlodic) [inline start.py:246]: this is cool but you have copy/pasted the code. Could you please optimize it with an external function?

**Comment by mlodic** [comment]
you would also need to resolve merge conflicts before going forward

**Comment by siddhanthrathod** [comment]
Okay I'll do that

File: start.py (hunk 1 above)
Comment (siddhanthrathod) [reply start.py:246]: Yes I'll do that

File: start.py [hunk 2]
```diff
@@ -222,6 +223,10 @@ def start():
         "--project-directory",
         "docker",
     ]
+    # for docker-compose v2
+    try:subprocess.run(base_command[0],stdout=subprocess.DEVNULL,stderr=subprocess.DEVNULL,check=True)
```
Comment (mlodic) [inline start.py:227]: please configure and use linters like stated in the documentation

**Review by mlodic — APPROVED** [review]

**Comment by mlodic** [comment]
ty!

**Pull request merged**